lib/tilex_web/controllers/auth_controller.ex

```diff
@@ -32,21 +32,36 @@ defmodule TilexWeb.AuthController do
     |> redirect(to: "/")
   end
 
-  defp authenticate(%{info: info}) do
-    email = Map.get(info, :email)
-    name = Developer.format_username(Map.get(info, :name))
-
-    case String.match?(email, ~r/@#{Application.get_env(:tilex, :hosted_domain)}$/) do
-      true ->
+  defp authenticate(%{info: %{email: email, name: name}}) when is_binary(name) do
```

Comment:
We could guard email as well as it appears we were doing so [here](https://github.com/hashrocket/tilex/pull/268/files#diff-208e6bebf49bf89eb76c14c38e24248eL19)

Reply:
>The user's full name, in a displayable form. Might be provided when:
The request scope included the string "profile"
The ID token is returned from a token refresh
When name claims are present, you can use them to update your app's user records. Note that this claim is never guaranteed to be present.